terraform/modules/task-definitions/content-store/main.tf

```diff
@@ -64,6 +64,7 @@ module "task_definition" {
         { "name" : "PLEK_SERVICE_PUBLISHING_API_URI", "value" : "http://publishing-api-web.${var.service_discovery_namespace_name}" },
         { "name" : "PLEK_SERVICE_ROUTER_API_URI", "value" : "http://${var.router_api_hostname_prefix}router-api.${var.service_discovery_namespace_name}" },
         { "name" : "PLEK_SERVICE_RUMMAGER_URI", "value" : "" },
+        { "name" : "PLEK_SERVICE_SIGNON_URI", "value" : "https://signon-ecs.${var.govuk_app_domain_external}" },
```

Comment:
I'm guessing the change of user-facing name is so that the old and new can coexist? This is probably something we'll need to think about a bit more in terms of how we'd switch things over. Ok for now though.

Reply:
Yes, I don't think we've worked this out yet.